controllers/comments.controller.js

```diff
@@ -0,0 +1,12 @@
+const { insertComment } = require('../models/comments.model')
+
+exports.postComment = (req, res, next) => {
+	const { article_id } = req.params
+	if (article_id && typeof req.body.body === 'string') {
+		insertComment(article_id, req.body)
+			.then(({ rows }) => {
+				res.status(201).send({ comment: rows[0] })
```

Comment:
Rows[0] shouldn't really be here - you should be sending this back from the model